README.CentOS7.md

```diff
@@ -43,7 +43,7 @@ MySQL is not recommended/supported on CentOS, see [paragraph 2 here](https://www
 *Note:* LORIS developers (those NOT working with a .zip release codebase) should skip steps relating to hosting their database locally. Contact your system administrator for database credentials.
 
 ```bash
-sudo yum install MariaDB-client MariaDB-server
+sudo yum install mariadb mariadb-server
```

Comment:
Package that come with CentOS 7 is MariaDB5 5.x, not 10.3
Need to use Package from MariaDB repo, so capital case package name is Ok. 
Need to add MariaDB repo like it is done for `remi` in the PHP section.

Reply:
@PapillonMcGill I'm a little confused on why `sudo yum install mariadb mariadb-server` wouldn't be acceptable? When I run yum info on mariadb I get this output:
```
$ sudo yum info mariadb
Loaded plugins: fastestmirror, langpacks
Loading mirror speeds from cached hostfile
 * base: mirror.centos.serverforge.org
 * epel: mirror.dst.ca
 * extras: mirror.centos.serverforge.org
 * remi-safe: mirror.bebout.net
 * updates: mirror.centos.serverforge.org
Installed Packages
Name        : mariadb
Arch        : x86_64
Epoch       : 1
Version     : 5.5.64
Release     : 1.el7
Size        : 49 M
Repo        : installed
From repo   : base
Summary     : A community developed branch of MySQL
URL         : http://mariadb.org
License     : GPLv2 with exceptions and LGPLv2 and BSD
Description : MariaDB is a community developed branch of MySQL.
            : MariaDB is a multi-user, multi-threaded SQL database server.
            : It is a client/server implementation consisting of a server daemon
            : (mysqld) and many different client programs and libraries. The
            : base package contains the standard MariaDB/MySQL client programs
            : and generic MySQL files.
```
and yum info on MariaDB-client returns this output:
```
$ sudo yum info MariaDB-client
Loaded plugins: fastestmirror, langpacks
Loading mirror speeds from cached hostfile
 * base: centos.mirror.globo.tech
 * epel: mirror.dst.ca
 * extras: centos.mirror.globo.tech
 * remi-safe: repo1.ash.innoscale.net
 * updates: centos.mirror.ca.planethoster.net
Error: No matching Packages to list
```
Seems mariadb comes from base repo and is that bad?

Reply:
base repo as MariaDB 5.x not 10.3

Reply:
@PapillonMcGill I added the "MariaDB Package Repository Setup Script" as referenced here: 
https://mariadb.com/kb/en/library/yum/
It sets the Repository to the official repo "mariadb-main", thoughts?

Reply:
On that same page, in the `Using the MariaDB Repository Configuration Tool` section, it show how to manually select 10.3.

```
For example, if you wanted to use the repository to install MariaDB 10.3 on CentOS 7, then you could use the following yum repository configuration in /etc/yum.repos.d/MariaDB.repo:

[mariadb]
name = MariaDB
baseurl = http://yum.mariadb.org/10.3/centos7-amd64
gpgkey=https://yum.mariadb.org/RPM-GPG-KEY-MariaDB
gpgcheck=1
```
This will use 10.3 repo for CentOS7

Reply:
@PapillonMcGill here is what shows after running the MariaDB Package Repository Setup Script script. Seems like the versions are up to date without me making an extra file?

```
$ sudo yum info MariaDB-client MariaDB-server
Loaded plugins: fastestmirror, langpacks
Loading mirror speeds from cached hostfile
 * base: centos.mirror.globo.tech
 * epel: d2lzkl7pfhq30w.cloudfront.net
 * extras: centos.mirror.globo.tech
 * remi-safe: repo1.ash.innoscale.net
 * updates: centos.mirror.ca.planethoster.net
Installed Packages
Name        : MariaDB-client
Arch        : x86_64
Version     : 10.4.8
Release     : 1.el7.centos
Size        : 60 M
Repo        : installed
From repo   : mariadb-main
Summary     : MariaDB: a very fast and robust SQL database server
URL         : http://mariadb.org
License     : GPLv2
Description : MariaDB: a very fast and robust SQL database server
            : 
            : It is GPL v2 licensed, which means you can use the it free of
            : charge under the conditions of the GNU General Public License
            : Version 2 (http://www.gnu.org/licenses/).
            : 
            : MariaDB documentation can be found at https://mariadb.com/kb
            : MariaDB bug reports should be submitted through
            : https://jira.mariadb.org

Name        : MariaDB-server
Arch        : x86_64
Version     : 10.4.8
Release     : 1.el7.centos
Size        : 126 M
Repo        : installed
From repo   : mariadb-main
Summary     : MariaDB: a very fast and robust SQL database server
URL         : http://mariadb.org
License     : GPLv2
Description : MariaDB: a very fast and robust SQL database server
            : 
            : It is GPL v2 licensed, which means you can use the it free of
            : charge under the conditions of the GNU General Public License
            : Version 2 (http://www.gnu.org/licenses/).
            : 
            : MariaDB documentation can be found at https://mariadb.com/kb
            : MariaDB bug reports should be submitted through
            : https://jira.mariadb.org

Available Packages
Name        : mariadb-server
Arch        : x86_64
Epoch       : 1
Version     : 5.5.64
Release     : 1.el7
Size        : 11 M
Repo        : base/7/x86_64
Summary     : The MariaDB server and related files
URL         : http://mariadb.org
License     : GPLv2 with exceptions and LGPLv2 and BSD
Description : MariaDB is a multi-user, multi-threaded SQL database server. It is
            : a client/server implementation consisting of a server daemon
            : (mysqld) and many different client programs and libraries. This
            : package contains the MariaDB server and some accompanying files
            : and directories. MariaDB is a community developed branch of MySQL.
```

Reply:
That installed MariaDB 10.4, not 10.3 Might still be ok, just that we don't test using 10.4

You can check in `/etc/ỳum.repos.d/MariaDB.repo` and check for `baseurl` line. It should be something like `latest` or `stable` in place of `10.3`

Reply:
You can also show how to import the key
`sudo rpm --import https://yum.mariadb.org/RPM-GPG-KEY-MariaDB`

Reply:
You're right @PapillonMcGill the contents are:
```
[mariadb-main]
name = MariaDB Server
baseurl = https://downloads.mariadb.com/MariaDB/mariadb-10.4/yum/rhel/$releasev$
gpgkey = file:///etc/pki/rpm-gpg/MariaDB-Server-GPG-KEY
gpgcheck = 1
enabled = 1
```
I'll add instructions to create the file.